TypewiseAlert.Test/TypewiseAlertTest.cs

```diff
@@ -8,8 +9,34 @@ public class TypewiseAlertTest
     [Fact]
     public void InfersBreachAsPerLimits()
     {
-      Assert.True(TypewiseAlert.inferBreach(12, 20, 30) ==
-        TypewiseAlert.BreachType.TOO_LOW);
+      Assert.True(TypewiseAlert.inferBreach(12, 20, 30) is LowBreachType);
+      Assert.True(TypewiseAlert.inferBreach(35, 20, 30) is HighBreachType);
+      Assert.True(TypewiseAlert.inferBreach(25, 20, 30) is NormalBreachType);
+      Assert.True(TypewiseAlert.classifyTemperatureBreach(new PassiveCooling(), 50) is HighBreachType);
```

Comment:
It is a good practice to include all permutation and combinations of input values i.e. type of cooling and values per cooling (out of range, in range, and boundary values). But if the combinations are very huge then select distinctive sets of tests.